app/models/note.rb

```diff
@@ -0,0 +1,21 @@
+class Note < ActiveRecord::Base
+  include ::Translatable
+
+  # create_table :notes do |t|
+  #   t.references :notable, polymorphic: true, index: true
+  #   t.references :person, index: true
+  #   t.timestamps
+
+
+  belongs_to :notable, polymorphic: true
+  belongs_to :person
```

Comment:
Is this person the author/creator of the note? If so, it might make more sense to stick to the more specific terminology, especially since people are also notable.